docs/bit-dot-dev/authentication.md

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
when renaming / moving files can we add to the netlify redirects file in this repo so that we don't get 404s from any old links that may have been set up etc